modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-node-affinity.adoc
+// * nodes/scheduling/nodes-scheduler-pod-affinity.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+ifeval::["{context}" == "nodes-scheduler-pod-affinity"]
+:pod:
+endif::[]
+ifeval::["{context}" == "nodes-scheduler-node-affinity"]
+:node:
+endif::[]
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-overriding-operator-pod-affinity_{context}"]
+
+ifdef::olm[]
+= Controlling where an Operator is installed
+endif::olm[]
+
+ifdef::pod[]
+= Using pod affinity and anti-affinity to control where an Operator is installed
+endif::pod[]
+
+ifdef::node[]
+= Using node affinity to control where an Operator is installed
+endif::node[]
+
+By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.
+
+The following examples describe situations where you might want to schedule an Operator pod to a specific node or set of nodes:
+
+* If an Operator requires a particular architecture, such as `amd64` or `arm64`
+* If an Operator requires a particular operating system, such as Linux or Windows
+* If you want Operators that work together scheduled on the same host or on hosts located on the same rack
+* If you want Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues
+
+ifdef::olm[]
+You can control where an Operator pod is installed by adding node affinity, pod affinity, or pod anti-affinity constraints to the Operator's `Subscription` object. Node affinity is a set of rules used by the scheduler to determine where a pod can be placed. Pod affinity enables you to ensure that pods are scheduled to the same node. Pod anti-affinity allows you to prevent a pod from being scheduled on a node. 
```

Comment:
Pod affinity enables you to ensure that **related** pods are scheduled to the same node